docs/general/how-ra-works.md

```diff
@@ -0,0 +1,45 @@
+# How RetroAchievements Works
+
+RetroAchievements (RA) provides users the ability to earn achievements in retro games from an RA achievement set.  It awards achievements by comparing a game’s memory, henceforth referred to as RAM, to achievement code written by an RA developer.  Achievement code, also known as logic, is a list of memory conditions chosen by the developer, that when all are simultaneously true on a single frame, will award an achievement.
+
+## What Is an Achievement Set?
+
+An achievement set is the compilation of individual achievements, Rich Presence and leaderboards for a particular game.  Achievements are comprised of achievement logic, a title, a description, a point value and a badge.  Additionally, a set must contain a Rich Presence script which provides site users information on where active players are in a game.  Sets may have leaderboards that track how well players perform in certain things in a game such as how quickly they can beat stages, how many points they can score and many other things.  RA users interface with all these components and subcomponents.  Each one is a vital part of the player experience.
+
+## What Is Achievement Code and How Is It Written?
+
+RA code is a custom text-based language that is interpreted by RA supporting emulators. This code is constructed by developers with one or two achievement development tools.  Developers may use whichever tool they prefer. The Achievement Editor in the RAIntegration toolkit requires no prior knowledge of coding. On the other hand, RATools is a standalone executable designed for developers who find it more comfortable to write in a scripting language to develop their achievement set. Both the Achievement Editor and RATools produce similar code that is used for all achievements, leaderboards and Rich Presence.
+
+### How Is Achievement Code Processed?
+
+A game’s RAM is its most basic form.  RAM is simply the game's memory addresses and their respective values at any given time.  When processed by a console or emulator, the RAM can be used to recognize what is happening within the game.  Games are processed in a series of frames, generally 60, 50, 30 or 25 per second depending on console and format.  RA processes all unearned achievement logic in a set every frame.  Because of this, developers can create logic such that when all of its conditions are simultaneously true, identify something uniquely precise happening in a game and award an achievement, activate a leaderboard and provide accurate Rich Presence.  It is an achievement developer’s job to understand enough of a game's RAM to be able to construct this logic.
```

Comment:
I would exclude the first sentence - it's arguable the game's most basic form is the ROM. The second sentence, with a slight tweak, can stand on its own:

> RAM is where a game stores its memory at any given time.

Reply:
Changed as recommended.